UE/Tests/Application/Ports/BtsPortTestSuite.cpp

```diff
@@ -11,203 +11,212 @@
 
 namespace ue
 {
-using namespace ::testing;
+    using namespace ::testing;
```

Comment:
Jak dla mnie te spacje tutaj nie potrzebne, namespace jest na cały plik i tak. Tak samo chyba w BTS Port było. I lepiej zostawić wcześniejsza wersję, bo chyba i tak większość plików w projekcie ma taką strukturę, że nie ma spacji w klamrach namespecowych, ale te spacje to z tego formater wynikają?

Reply:
W sumie jak chcesz dodać spację do projektu przez tego formatera to zrób na to osobny branch, bo trochę źle się czyta zmieniony kod przez te dodawanie spacji :(

Reply:
XD